coursedata/adventures/de.yaml

```
@@ -603,164 +609,195 @@ adventures:
                     ```
 
 
-                start_code: "forward 50\nturn left"
+                start_code: |
+                    forward 50
+                    turn left
             2:
                 story_text: |
-                    In level 1 the turtle could only turn left or right. That is a bit boring!
-                    In level 2 he can point his nose in all directions.
+                    In Level 1 kann sich die Schildkröte nur nach links und rechts drehen.
+                    Das ist etwas langweilig!
+                    In Level 2 kann sie ihre Nase in alle Richtungen strecken.
 
-                    Use 90 to turn a quarter. We call this degrees. A full turn is 360 degrees.
-                    Can you make a figure with this code? Maybe a triangle or a circle?
+                    Nutze 90, um wieder eine viertel Drehung zu machen.
+                    Diese Angabe ist in "Grad (°)".
+                    Eine volle Drehung um sich selbst sind 360° (360 Grad).
+                    Kannst Du mit diesem Code eine bestimmte Form zeichnen?
+                    Ein Dreieck zum Beispiel, oder einen Kreis?
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
 
                     ```
-                    print Drawing figures
-                    angle is 90
-                    turn angle
+                    print Zeichne Formen
+                    Winkel is 90
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
                     ```
                 start_code: |
-                    print Turtle race!
-                    hoek is 90
-                    turn hoek
+                    print Schildkrötenrennen!
+                    Winkel is 90
+                    turn Winkel
                     forward 25
             3:
                 story_text: |
-                    ## Searching turtle
-                    In this level you can use use `random` with the drawing turtle. A random choice makes the turtle walk a different path each time.
-                    Use `at random` to choose a value from a list. You can copy and paste lines 2 and 3 to create a longer random path.
+                    ## Suchende Schildkröte
+                    In diesem Level kannst Du den Befehl `random` nutzen, während die Schildkröte etwas zeichnet.
+                    Durch eine zufällige Wahl sieht die Zeichnung jedes Mal anders aus.
```

If we want it closer to the English version:
```suggestion
                    Durch eine zufällige Wahl nimmt die Schildkröte jedes Mal einen anderen Weg.
```
(Though your version is fine, too.)